cordova-plugin-appcenter-shared/src/android/AppCenterSharedPlugin.java

```diff
@@ -22,6 +22,21 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
             AppCenterUtils.sendUUIDPluginResultFromFuture(AppCenter.getInstallId(), callbackContext);
             return true;
         }
+        if (action.equals("setUserId")) {
+            String userId = args.getString(0);
+            AppCenterUtils.sendVoidPluginResultFromFuture(setUserId(userId), callbackContext);
+            return true;
+        }
         return false;
     }
+
+    private synchronized AppCenterFuture<Void> setUserId(final String userId) {
+        final DefaultAppCenterFuture<Void> future = new DefaultAppCenterFuture<>();
+        final android.os.Handler handler = new android.os.Handler();
```

Comment:
Use import instead of `android.os.` prefix

Reply:
Fixed, thanks for the review!